src/test/java/ProjectOcean/Model/CourseTests.java

```diff
@@ -10,13 +10,29 @@
 
     @Test
     public void courseUniqueIdTest() {
-        Course course1 = new Course("DAT017","Maskinorienterad programmering", 7.5f, 1, "Rolf Söderström", "Tenta", "Svenska", new ArrayList<>(), "www.google.com", "Lorem Ipsum");
-        Course course2 = new Course("DAT017","Maskinorienterad programmering", 7.5f, 1, "Rolf Söderström", "Tenta", "Svenska", new ArrayList<>(), "www.google.com", "Lorem Ipsum");
+        CourseFactory.SetStudyPeriod(1);
+        CourseFactory.SetCourseInfo("DAT017","Maskinorienterad programmering", 7.5f);
+        CourseFactory.SetCourseDetails(new ArrayList<>(), "www.google.com", "Lorem Ipsum");
+        CourseFactory.SetCourseAccessibility("Rolf Söderström", "Tenta", "Svenska");
+        CourseFactory.CreateCourse();
 
-        //Checks that UUID references are different
-        Assert.assertFalse(course1.getId()==course2.getId());
-        //Checks that UUID values are different
-        Assert.assertFalse(course1.getIdToString().equals(course2.getIdToString()));
+        CourseFactory.SetStudyPeriod(1);
+        CourseFactory.SetCourseInfo("DAT017","Maskinorienterad programmering", 7.5f);
+        CourseFactory.SetCourseDetails(new ArrayList<>(), "www.google.com", "Lorem Ipsum");
+        CourseFactory.SetCourseAccessibility("Rolf Söderström", "Tenta", "Svenska");
+        ICourse course1 = CourseFactory.CreateCourse();
+
+        CourseFactory.SetStudyPeriod(1);
+        CourseFactory.SetCourseInfo("DAT017","Maskinorienterad programmering", 7.5f);
+        CourseFactory.SetCourseDetails(new ArrayList<>(), "www.google.com", "Lorem Ipsum");
+        CourseFactory.SetCourseAccessibility("Rolf Söderström", "Tenta", "Svenska");
+        ICourse course2 = CourseFactory.CreateCourse();
+
+        //Checks that ICourse references are different
+        Assert.assertFalse(course1 == course2);
+
+        //Checks that ICourse values are different
+        Assert.assertFalse(course1.equals(course2));
 
     }
```

Comment:
What is this method even for anymore? We have no ID... All this is doing is basically testing if 2 different objects have different references 2 times. Remove it for easy fix